lib/ui/common/dasher_tweets_list.dart

```diff
@@ -1,64 +1,64 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_dasher/common/model/tweet.dart';
 import 'package:flutter_dasher/ui/common/dasher_tweet.dart';
-import 'package:flutter_dasher/ui/dashboard/presenter/feed_request_presenter.dart';
+import 'package:flutter_dasher/ui/dashboard/presenter/feed.dart';
 import 'package:hooks_riverpod/hooks_riverpod.dart';
 
 class DasherTweetsList extends ConsumerWidget {
-  const DasherTweetsList({
-    Key? key,
-  }) : super(key: key);
+  const DasherTweetsList({super.key});
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    final _presenter = ref.watch(feedRequestPresenter);
+    final feed = ref.watch(feedProvider);
 
     return Center(
-      child: _presenter.state.maybeWhen(
-        success: (feed) => _TweetsList(
-          feed: feed,
-        ),
-        initial: () => const CircularProgressIndicator(),
-        loading: (feed) {
-          if (feed == null) {
-            return const CircularProgressIndicator();
-          } else {
-            return _TweetsList(
-              feed: feed,
-            );
-          }
-        },
-        failure: (e) => Text('Error occurred $e'),
-        orElse: () => const CircularProgressIndicator(),
-      ),
+      child: switch (feed) {
+        AsyncData(:final value) => _TweetsList(
+            onRefresh: () => ref.refresh(feedProvider.future),
+            tweets: value,
+          ),
+        AsyncError(:final error, isLoading: false) => Column(
```

Comment:
I like this is loading false matching, and then default to loading. Can we also mention this in handbook?